test/datasets_utils.py

```diff
@@ -728,6 +727,10 @@ def check_transforms_v2_wrapper_spawn(dataset):
         assert tree_any(
             lambda item: isinstance(item, (tv_tensors.Image, tv_tensors.Video, PIL.Image.Image)), wrapped_sample
         )
+        from torchvision.datasets import VOCDetection
+
+        if isinstance(dataset, VOCDetection):
+            assert wrapped_sample[0][0].size == (321, 123)
```

Comment:
the point of this test is just to show that the fix works. It obviously won't stay as-is.

We should really add transforms to this test though. @pmeier Do you have any suggestion on how you'd prefer to do that? Otherwise, I'll kinda just do something similar for all datasets, i.e. just check that the images in `wrapped_sample` are of a specific expected size.